src/std/bytes.c

```diff
@@ -143,7 +145,9 @@ HL_PRIM int hl_bytes_rfind( vbyte *where, int len, vbyte *which, int wlen ) {
 }
 
 HL_PRIM void hl_bytes_fill( vbyte *bytes, int pos, int len, int value ) {
-	memset(bytes+pos,value,len);
+	if(bytes) {
```

Comment:
So this will now fail silently? If this check is performed, might as well emit a warning or something?

Reply:
That happened when I was testing the changes with some openfl templates. It seems that somewhere it passes NULL to `hl_bytes_fill` and `hl_bytes_blit`.